Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -356,53 +347,64 @@ - (void)configureViews
             }
 
             if (self.hasBodyMediaWithThumbnail) {
-
                 // The "body media" view casts a shadow "downward" onto adjacent views,
                 // so we use a "proxy" view to take its place within the v-stack
                 // view and then insert the body media view above its proxy so that
                 // it floats above the other content of the bubble view.
 
-                UIView *bodyProxyView = [UIView new];
-                [self.stackView addArrangedSubview:bodyProxyView];
+                UIView *proxyView = [UIView new];
```

Comment:
proxy for what?

Reply:
I guess the comment above isn't clear?  It's a layout proxy.  It takes the place of a shadow-casting view in a stack view's layout so that the shadow-casting view can be added _above_ the stack view and cast shadows onto its contents.